src/bin/cargo/commands/info.rs

```diff
@@ -0,0 +1,39 @@
+use cargo::ops::cargo_info::info;
+use cargo::util::command_prelude::*;
+use cargo_util_schemas::core::PackageIdSpec;
+
+pub fn cli() -> Command {
+    Command::new("info")
+        .about("Display information about a package in the registry")
+        .arg(
+            Arg::new("package")
```

Comment:
I am not sure whether we need to use `crate` here.

Reply:
We do inspect the crate, but we support package ID as a specification. So I don't know which term we should use here.

Reply:
Are you referring to `Arg::new(name)`?  The `name` is only for internal processing purposes though I'd be consistent with other SPECs.